RevenueCatUI/Helpers/Variables.swift

```diff
@@ -19,45 +20,50 @@ protocol VariableDataProvider {
 
 }
 
+struct VariableMatch {
```

Comment:
This can be `private`. Maybe also move it inside `VariableHandler`?

```
private extension VariableHandler {
  
    struct Match {
        ...
    }

}
```

Then `VariableHandler` can just refer to `Match`.

Reply:
good idea! Updated